rudder-agent/debian/rules

```diff
@@ -83,7 +83,8 @@ binary-arch: install
 #	dh_installlogrotate
 #	dh_installmime
 #	dh_python
-	dh_installinit --no-start -R # restart after upgrade (instead of stop, upgrade, start)
+	dh_installinit --update-rcd-params=disable --no-start -R # restart after upgrade (instead of stop, upgrade, start)
+	dh_installinit --name=rudder --no-start -R # restart after upgrade (instead of stop, upgrade, start)
```

Comment:
We avoid at all costs starting/stopping scripts during install and upgrades, to be consistent between all our packages (and prevent os-dependent surprises). Post installation / migration scripts and the agent take care of restarting what needs restarting when needed !